### PR TITLE
Add additional use cases and examples to intro

### LINKED ISSUE #181: [spec] Add additional examples
The examples section currently only has government ID and some TBD text.

```html
    <p id="credential-type-examples">
      There are many types of digital credential that can be conveyed using
      this API. <dfn data-lt="credential type examples">Examples of these
      types</dfn> include:
    </p>
    <ul>
      <li>a driver's license or identity card issued by a government
      institution
      </li>
      <li>TBW: other credential types
      </li>
    </ul>
```

Update to include:

- employee ID
- federated sign in
- verified claim (phone, email)

### PR DESCRIPTION
**Pull request by timcappalli**
Closes #181

Adds additional use cases to the intro section.

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/digital-credentials/pull/186.html" title="Last updated on Oct 28, 2024, 10:01 PM UTC (2529b97)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/digital-credentials/186/3dfefe5...2529b97.html" title="Last updated on Oct 28, 2024, 10:01 PM UTC (2529b97)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,15 +68,22 @@ <h2>
...
       </li>
-      <li>TBW: other credential types
+      <li>a travel authorization document issued by an embassy or consulate
+      </li>
+      <li>a proof of employment issued by a private organization
+      </li>
+      <li>a proof of education or professional training issued by an institution
+      </li>
+      <li>and many other scenarios as described in 
+        <a href="https://www.w3.org/TR/vc-use-cases/">Verifiable Credentials Use Cases</a>
```
Comment (samuelgoto) [inline index.html:86]: These are great use cases. It is not perfectly clear to me whether all of these are going to be best served by this API, but I think is a great baseline to work from.

**Review by samuelgoto — APPROVED** [review]
Thanks for putting this together! I appreciate skipping the Social Login scenario (apart from the indirect reference from the VC use-cases doc), as we figure out what's the best way to go about it! LGTM!

**Review by leecam — APPROVED** [review]
LGTM

**Review by marcoscaceres — APPROVED** [review]
Cool, but might be good to keep examples matching reality.

**Pull request merged**